cot/Cargo.toml

```diff
@@ -72,10 +72,13 @@ url = { workspace = true, features = ["serde"] }
 
 [dev-dependencies]
 async-stream.workspace = true
+criterion = { workspace = true, features = ["async_tokio"] }
 fake.workspace = true
 fantoccini.workspace = true
 futures.workspace = true
 mockall.workspace = true
+reqwest = { version = "0.11", features = ["json"] }
+serde_urlencoded = "0.7"
```

Comment:
As per https://github.com/cot-rs/cot/blob/master/CONTRIBUTING.md#dependencies , please move the versions to a workspace-global `Cargo.toml` and use `dep.workspace = true` or `dep = { workspace = true, features = ... }` here.